tests/rptest/tests/cluster_config_test.py

```diff
@@ -1226,6 +1226,31 @@ def yamlize(input) -> str:
                     f"Config setting {key}={strval} should have been rejected")
                 assert False
 
+        # Check that the `get` command hits proper validation paths
+        unknown_examples = [
+            "panda_size", "panda_retention_ms", "panda_mutation_rate"
+        ]
+
+        for key in unknown_examples:
+            try:
+                self.rpk.cluster_config_get(key)
+            except RpkException as e:
+                pass
+            else:
+                self.logger.error(
+                    f"Config getting {key} should have been rejected")
+                assert False
```

Comment:
suggestion/fyi: This pattern is succinctly expressed by [expect_exception](https://github.com/redpanda-data/redpanda/blob/9d9a14844c14f583220c510122a509c6ec4f57bd/tests/rptest/util.py#L297-L309). For example: 

https://github.com/redpanda-data/redpanda/blob/9d9a14844c14f583220c510122a509c6ec4f57bd/tests/rptest/tests/rpk_role_test.py#L82-L84

Reply:
Changed these statements to use `expect_exception`/